lib/krb5/log.c

```diff
@@ -45,17 +45,60 @@ krb5_initlog(krb5_context context,
     return heim_initlog(context->hcontext, program, fac);
 }
 
+struct krb5_addlog_func_wrapper {
+	krb5_context context;
+	krb5_log_log_func_t log_func;
+	krb5_log_close_func_t close_func;
+	void *data;
+};
+
+static void HEIM_CALLCONV krb5_addlog_func_wrapper_log(heim_context hcontext,
```

Comment:
Sorry to be picky, could you please put `krb5_addlog_func_wrapper_log` on a new line?